src/widgets/CustomerRequisitionItemDetail.js

```diff
@@ -17,9 +17,11 @@ import { DARKER_GREY, WHITE, SUSSOL_ORANGE } from '../globalStyles';
 const CustomerRequisitionItemDetailsComponent = ({ item }) => (
   <View style={localStyles.container}>
     <PageInfo
+      titleTextAlign="right"
       titleColor={SUSSOL_ORANGE}
       infoColor={WHITE}
       columns={getPageInfoColumns('customerRequisitionItemDetail')(item)}
+      numberOfLines={2}
```

Comment:
Added number of lines and text align pass through to align the title of page info to the right when needed and multiple lines when needed. Default prop to be what the old hard coded values were.